formidling/src/main/java/no/nav/ung/sak/formidling/bestilling/BrevbestillingTask.java

```diff
@@ -165,7 +166,7 @@ private OpprettJournalpostRequest opprettJournalpostRequest(UUID brevbestillingU
             .journalpostType("UTGAAENDE")
             .avsenderMottaker(avsenderMottaker)
             .bruker(bruker)
-            .tema("OMS") //TODO endre for ung
+            .tema(OmrådeTema.UNG.getKode())
             .behandlingstema("ab0271")
```

Comment:
hva med `behandlingstema` ? ab0271 er for OMP. Vet du @ramrock93

Reply:
Det vet jeg ikke. Jeg antar at det er noe vi kanskje trenger også for dette temaet.